ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -49,7 +53,16 @@ def __init__(self, *args, **kwargs):
         self.metadata_collector = MetadataCollector(self._config, self.log)
         self.stats_collector = StatsCollector(self._config, self.send_metrics_from_properties, self.log)
 
+        self.queue_manager_process_pattern = None
+
+        self.check_initializations.append(self.parse_config)
+
     def check(self, _):
+        if not self.check_queue_manager_process():
+            self.log.debug('Process not found, skipping check run')
+            self.service_check(self.SERVICE_CHECK, self.WARNING, self._config.tags, hostname=self._config.hostname)
```

Comment:
Would WARNING or UNKNOWN be better?

Reply:
I think unknown makes more sense because it is not intrinsically wrong as a warning would be, we just dont want it to be "no data"

Reply:
It should probably send unknowns too for the queue manager service check

Reply:
In further discussions with Jose maybe we should not send the can_connect service check at all because all instances would have the same tags `self.service_check(self.SERVICE_CHECK, AgentCheck.OK, self._config.tags, hostname=self._config.hostname)` so effectively we are submitting duplicate service tags.